tests/components/autarco/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,22 @@
+# serializer version: 1
+# name: test_entry_diagnostics
+  dict({
+    'config_entry_data': dict({
+      'email': '**REDACTED**',
+      'password': '**REDACTED**',
+    }),
```

Comment:
Why do we even log the config entry in this case?

Reply:
I assumed this was "default" stuff to put in, but since everything is redacted it can actually be removed 😅